src/libs/nav2_robot/include/nav2_robot/ros_robot.hpp

```diff
@@ -37,11 +38,12 @@ class RosRobot : public Robot
 
   void enterSafeState() override;
 
+  urdf::Model model_;
   geometry_msgs::msg::PoseWithCovarianceStamped::SharedPtr getCurrentPose();
 
 protected:
   rclcpp::Node * node_;
-
+  const char * urdf_file_;
```

Comment:
I'd prefer to use std::string instead of char *. You can always get the char * using urdf_file_.c_str().

Reply:
Fixed.